src/features/movies/MovieList/MovieTile/index.js

```diff
@@ -0,0 +1,165 @@
+import styled from "styled-components";
+
+export const MovieTile = styled.div`
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    padding: 16px;
+    width: 324px;
+    height: 650px;
+    background-color: ${({ theme }) => theme.colors.white};
+    box-shadow: ${({ theme }) => theme.boxShadow};
+    border-radius: ${({ theme }) => theme.borderRadiusSmall};
+
+    @media (max-width: ${({ theme }) => theme.breakpoints.tablet}){
+        display: flex;
+        flex-direction: row;
+        justify-content: space-between;
+        padding: 16px;
+        width: 288px;
+        height: 201px;
+        background-color: ${({ theme }) => theme.colors.white};
+        box-shadow: ${({ theme }) => theme.boxShadow};
+        border-radius: ${({ theme }) => theme.borderRadiusSmall};
+        
+    }
+  
+    @media (max-width: ${({ theme }) => theme.breakpoints.mobile}) {
+        display: flex;
+        flex-direction: row;
+        justify-content: space-between;
```

Comment:
Jeżeli dla większej rozdzielczości ustawiłeś te właściwości, to dla mniejszej już tego robić nie musisz. Tego typu przypadków jest znacznie więcej. Przejrzyj proszę kod, gdzie jeszcze występuje powielanie.

Reply:
Wszystko powinno już zostać usunięte co niepotrzebne